tests/test_output.py

```diff
@@ -93,11 +93,12 @@ def test_influxdb(self):
                 },
             ]
         )
+        backlogger._today_nanoseconds = MagicMock(side_effect=[23])
```

Comment:
How about [this approach](https://stackoverflow.com/questions/4481954/trying-to-mock-datetime-date-today-but-not-working/55187924#55187924)?

Reply:
I can't get it to work. Here's my change:
diff --git a/tests/test_output.py b/tests/test_output.py
index b36da60..aeafe9f 100644
--- a/tests/test_output.py
+++ b/tests/test_output.py
@@ -2,6 +2,9 @@ import os
 import sys
 import unittest
 from unittest.mock import MagicMock, call
+from unittest import mock, TestCase
+import datetime
+
 
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
 
@@ -18,7 +21,8 @@ class TestOutput(unittest.TestCase):
         }
         backlogger.data = data
 
-    def test_influxdb(self):
+    @mock.patch(f'{backlogger.__name__}.datetime', wraps=datetime)
+    def test_influxdb(self, mock_datetime):
         backlogger.json_rest = MagicMock(
             side_effect=[
                 {"issue_statuses": [{"name": "In Progress", "id": 2}]},
@@ -93,7 +97,8 @@ class TestOutput(unittest.TestCase):
                 },
             ]
         )
-        backlogger._today_nanoseconds = MagicMock(side_effect=[23])
+        mock_datetime.date.today.return_value = datetime.date(2019, 3, 15)
+        #backlogger._today_nanoseconds = MagicMock(side_effect=[23])
         self.assertEqual(
             backlogger.render_influxdb(backlogger.data),
             [
```
I get:
```
E                   AttributeError: module 'datetime' has no attribute 'strptime'

/usr/lib64/python3.10/unittest/mock.py:661: AttributeError
================================================================================================== short test summary info ==================================================================================================
FAILED tests/test_output.py::TestOutput::test_influxdb - AttributeError: module 'datetime' has no attribute 'strptime'
```

Reply:
Ack. I tried it here but also couldn't fix that. Let's leave it for now.